job_spec/INSAR_ISCE.yml

```diff
@@ -37,6 +37,7 @@ INSAR_ISCE:
           - HRRR
           - HRES
           - GMAO
+          - ERA5
```

Comment:
Might actually be good to include *all* of these:
https://github.com/dbekaert/RAiDER/blob/develop/tools/RAiDER/models/allowed.py

Reply:
@jhkennedy Sweet. Added!